Sources/MapboxMaps/Foundation/ObservableValue.swift

```diff
@@ -0,0 +1,43 @@
+internal final class ObservableValue<Value> where Value: Equatable {
```

Comment:
This was mostly just a refactor, but the amount of changes were significant enough for it to show as a new file.